.github/workflows/.ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         run: pytest -c test/pytest.ini test/unit/ --cov
 
   e2e-tests:
-    needs: [unit-tests]
+    needs: [ unit-tests ]
```

Comment:
Have you got a different formatter to me? Mine just removes these spaces again. Its only this line and 94 that conflict with mine. I personally prefer not having spaces like this.

Reply:
PyCharm's formatter, and I think you use VS Code?